RealmModule/LocalFeedImageRealm.swift

```diff
@@ -0,0 +1,54 @@
+//
+//  LocalFeedImageRealm.swift
+//  FeedStoreChallenge
+//
+//  Created by Carlos Damiao on 16/01/2021.
+//  Copyright © 2021 Essential Developer. All rights reserved.
+//
+
+import Foundation
+import RealmSwift
+
+public class LocalFeedImageRealm: Object {
+	@objc dynamic var id = ""
+	@objc dynamic var _description = ""
+	@objc dynamic var location = ""
```

Comment:
`location` should be optional:

```suggestion
	@objc dynamic var location: String? = nil
```